server/celery-queue/tasks.py

```diff
@@ -42,40 +76,45 @@ def write_to_temp_file(name:str, data:str, folder:str):
         f.write(data)
     return path
 
-# Solve using downloaded flask files - not strings
-@celery.task(name='tasks.solve')
-def solve(domain_url: str, problem_url: str, solver: str) -> str:
-    tmpfolder = tempfile.mkdtemp()
+# The solve endpoint is replaced by the runpackage completely? So I have commented the following code.
```

Comment:
This was here for backwards compatibility to the old solver. I think we should either (1) maintain that capability; or (2) have an informative error (not just a server 500 :P) that points the users to the "new way of doing things". In particular, we should have a "change your old solve call to <this> if you want the same behaviour".

Reply:
This would only work if the solver passed as an argument is currently installed in the server. Looking at the API for Previous plugins, the API didn't specify a solver right? If we want to have backward compatibility we should have a default planner.

Ideally, we should list the existing Solve endpoints accepted by the workers. Feature request?

Reply:
I think a breaking change would be ok...and ya, would need to be an endpoint that doesn't require the solver to be specified. Just something that responds to the URLs currently advertised on solver.planning.domains. Then an error status with a pointer to a page on converting to the new server.

As for an endpoint to list the existing functionality, don't we already have that?

Reply:
Right, yes, we can do this with localhost:5001/package